### ISSUE TITLE
Are you a fellow of openSource or be a part of a OpenSource community?

### ISSUE THREAD

**Issue by edovio** [edovio/OpenStudentBook#50]
Hello Fellow! I'm happy to ask you to help us to improve the content about "what is the Open Source and the Open Project" !!! Write here your feedback about the content regarding this topic.

here you can find the link of the content specific for you:
https://edovio.gitbooks.io/open-student-book/content/en/open_source_introduction.html (EN)
https://edovio.gitbooks.io/open-student-book/content/it/open_source_introduction.html (IT)

**Comment by jvallera** [comment]
Hi @edovio loving the gitbook! wondering what you think about adding "licensing" to the first sentence or two in this section. For example, software is not technically open source until a license is assigned to it. "computer software with its source code made available with a license in which the copyright holder provides the rights to study, change, and distribute the software to anyone and for any purpose." - from wikimedia

**Comment by edovio** [comment]
yeah good point of view! Thanks for sharing @jvallera, So we can evaluate to change 
this phrase: In Information Technology a software is referred to as "Open Source" when its source code is publicly available for everyone to change or extend. In simpler words, the "recipe" for a work is shared, free and everyone can use it. 

With this: In Information Technology a software is referred as open source when have a license who assign it the rights to study, change, and distribute the software to anyone and for any purpose. In simpler words, the "recipe" for a work is shared, free and everyone can use it.

what do you think? :)

**Comment by jvallera** [comment]
Yes, that sounds good. I think you can also reference some resources that explain this in more detail so people can learn more. like this website: https://opensource.org/licenses or others that may be available on medium, github, etc.

Maybe you can find additional resources in Italian.
